pkg/channel/distributed/common/config/observability.go

```diff
@@ -32,6 +34,10 @@ import (
 	"knative.dev/pkg/profiling"
 )
 
+var ListenAndServeWrapper = func(srv *nethttp.Server) func() error { return srv.ListenAndServe }
+var StartWatcherWrapper = func(cmw *configmap.InformedWatcher, done <-chan struct{}) error { return cmw.Start(done) }
+var UpdateExporterWrapper = metrics.UpdateExporter
+
```

Comment:
can we put into a single `var` block and add a comment saying these are to facilitate unit testing?

Reply:
Yes; seems like a good idea.